src/pudl/metadata/fields.py

```diff
@@ -1144,6 +1144,11 @@
         "description": "Electricity demand (energy) within a given timeframe.",
         "unit": "MWh",
     },
+    "demand_agg_mwh": {
```

Comment:
Does this naming make sense? I know we use the `agg` tag in some column names, but not sure if we have formal conventions for when to do this.

Reply:
I don't think we need a separate column name here -- the variable that's being reported is still just `demand_mwh` (or maybe `demand_imputed_pudl_mwh`) -- the fact that it's aggregated up doesn't change the nature of the variable.  I think we should convey the nature of the geographic aggregation in the table level descriptions.

Reply:
Makes sense, I went with `demand_imputed_pudl_mwh` so it's still clear this is imputed data.